test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -69,57 +68,61 @@
 ;; TESTS
 
 (deftestseq node-queries
-  [[version endpoint] endpoints]
+  [[version endpoint] endpoints
+   method [:get :post]]
 
-  (let [{:keys [web1 web2 db puppet]} (store-example-nodes)]
+  (let [status-for-node' #(status-for-node method endpoint %)
```

Comment:
Yeah. I mention this just because I've seen it in a couple places. I'm fine with doing whatever seems right at the time.